packages/cosmic-swingset/src/chain-main.js

```diff
@@ -64,6 +64,35 @@ const toNumber = specimen => {
   return number;
 };
 
+/**
+ * A boot message consists of cosmosInitAction fields that are subject to
+ * consensus. See cosmosInitAction in {@link ../../../golang/cosmos/app/app.go}.
+ *
+ * @param {any} initAction
+ */
+const makeBootMsg = initAction => {
+  const {
+    type,
+    blockTime,
+    blockHeight,
+    chainID,
+    isBootstrap,
+    params,
+    supplyCoins,
+    upgradeDetails,
+  } = initAction;
+  return {
+    type,
+    blockTime,
+    blockHeight,
+    chainID,
+    isBootstrap,
```

Comment:
I don't see a reason why `isBootstrap` would be useful in the `bootMsg`. We can likely can drop it.